sandworm/Resources/NotPetya/SharpNP/packages/DllExport.1.7.4/.gitattributes

```diff
@@ -0,0 +1,7 @@
+
```

Comment:
Great point. I wasn't there for the code creation on this one. Let's leave this since it's small (don't want to break anything) and keep this use case in mind for the next evaluation merged. I am a fan of smaller PRs and clear instructions on builds.